delivery_postlogistics_dangerous_goods/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Postlogistics Shipping Dangerous Goods",
+    "summary": "Declare dangerous goods when generating postlogistics labels",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "maintainer": "Camptocamp",
+    "license": "AGPL-3",
+    "category": "Delivery",
+    "complexity": "normal",
+    "depends": [
+        # OCA/delivery-carrier
+        "delivery_postlogistics",
+        # OCA/community-data-files
+        "l10n_eu_product_adr",
+    ],
+    "website": "https://github.com/OCA/delivery-carrier",
+    "installable": True,
+    "auto_install": False,
```

Comment:
`auto_install` False is the default, so not really needed, here.
But does it not make sense to have it set to `True` ?

Reply:
@mmequignon ping

Reply:
Yeah, I don't know in what case we should enable auto-install or not

Reply:
With `auto_install` set to `True` your module will be automatically installed when both its dependencies are installed. Which in my opinion makes sense in this case.

Reply:
It makes sense to set it to `True` as this module doesn't break the base PostLogistics module.